tests/test_mesh.py

```diff
@@ -908,6 +873,36 @@ def test_nativetag_expand():
     m.clam_001 = NativeMeshTag(1, float)
     assert_array_equal(m.clam_001[:], 2.2)
 
+def test_nativetag_mult():
+    m = Mesh(structured=True, structured_coords=[[-1, 0, 1], [0, 1], [0,1]])
+    m.peach = NativeMeshTag(1, float)
+    m.peach[:] = [1.5, 2.5]
+    m.tangerine = NativeMeshTag(1, float) 
+    m.tangerine[:] = [5.0, 10.0]
+    assert_array_almost_equal(m.peach.mult(m.tangerine), [7.5, 25.0]) #multiplying two scalar-valued tags
+    
+    m.plum = NativeMeshTag(2, float)
+    m.plum[:] = [[5.0, 10.0], [15.0, 20.0]]
+    assert_array_almost_equal(m.peach.mult(m.plum), [[7.5, 25.0], [22.5, 50.0]]) #multiplying scalar-valued tag by vector-valued tag
```

Comment:
This is related to my comment in the `mult` function. I think that the expected value should be `[[7.5, 15 ], [37.5, 50]].`
Also, I think this test would fail if you had a `(2,3)` array for `plum.` Have you tried other `size` for `NativeMeshTag`?

Reply:
Okay, I think I understand. I'll see what solutions there are in this situation. Do the other cases look okay to you, or do they nee adjustment too?